crates/bevy_render/src/texture/dds.rs

```diff
@@ -45,7 +45,12 @@ pub fn dds_buffer_to_image(
         depth_or_array_layers,
     }
     .physical_size(texture_format);
-    image.texture_descriptor.mip_level_count = dds.get_num_mipmap_levels();
+    let mut mip_map_level = dds.get_num_mipmap_levels();
+    if mip_map_level == 0 {
+        warn!("Mipmap levels for texture are 0, bumping them to 1");
```

Comment:
Only logging during `#[cfg(debug_assertions)]` will improve performance on debug builds.
```suggestion
        #[cfg(debug_assertions)]
        warn!("Mipmap levels for texture are 0, bumping them to 1");
```

Reply:
We should use a `warn_once` to avoid spamming, and ideally try and include some information about which asset is producing this error if possible.